examples/ny-tlc-report-external-dependencies.yaml

```diff
@@ -25,9 +25,8 @@ spec:
           port: 9000
   sparkConf:
     spark.hadoop.fs.s3a.aws.credentials.provider: "org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider"
-    spark.hadoop.fs.s3a.path.style.access: "true"
```

Comment:
If we remove `spark.hadoop.fs.s3a.path.style.access: "true"` here thne we need to include it in the s3 definition e.g.
```
  s3bucket:
    inline:
      ...
      connection:
        inline:
          ...
          accessStyle: Path
```
(not tested)

Reply:
Done.